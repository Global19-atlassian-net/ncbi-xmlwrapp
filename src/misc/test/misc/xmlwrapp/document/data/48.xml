<?xml version="1.0"?>
<root>
    <carrot     a=  "one" g="two" z   = "three">red</carrot>
    <!-- comment -->
  <banana>yellow</banana><apple  z="zzz"     a = "aaa"></apple><zorro>
value</zorro>
</root>

