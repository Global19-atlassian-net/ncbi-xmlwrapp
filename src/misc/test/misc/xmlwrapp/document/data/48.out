<?xml version="1.0"?>
<root>
    <carrot a="one" g="two" z="three">red</carrot>
    <!-- comment -->
  <banana>yellow</banana><apple a="aaa" z="zzz"/><zorro>
value</zorro>
</root>
